samples/snippets/csharp/VS_Snippets_CLR_System/system.String.Format/cs/starting1.cs

```diff
@@ -41,18 +41,18 @@ private static void Snippet33()
        // <Snippet33>
        int[] years = { 2013, 2014, 2015 };
        int[] population = { 1025632, 1105967, 1148203 };
-       String s = String.Format("{0,6} {1,15}\n\n", "Year", "Population");
-       for(int index = 0; index < years.Length; index++)
-          s += String.Format("{0,6} {1,15:N0}\n",
-                             years[index], population[index]);
+       StringBuilder myStringBuilder = new StringBuilder();
+       myStringBuilder.Append(String.Format("{0,6} {1,15}\n\n", "Year", "Population"));
```

Comment:
In addition to the C# code, there is also Visual Basic and C++/CLI code that needs to have parallel changes. Can you make those, @gernotpokorny? The relevant files are: 
**For VB**, https://github.com/dotnet/docs/blob/master/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.String.Format/vb/starting1.vb.

The code changes:
   ```
   Dim sb As New StringBuilder()
   sb.Append($"{"Year",6} {"Population",15}{vbCrLf}{vbCrLf}")
   For index = 0 To years.Length - 1
      sb.Append($"{years(index),6} {population(index),15:N0}{vbCrLf}")
   Next
   ``` 
Also, there should be an `Imports System.Text` statement at the beginning of the file, and the variable name in the `Console.WriteLine` method call should be changed,

**For C++**, https://github.com/dotnet/docs/blob/master/samples/snippets/cpp/VS_Snippets_CLR_System/system.String.Format/cpp/starting1.cpp

The code changes: 
```
StringBuilder^ sb = gcnew StringBuilder();
for(int index = 0; index < years->Length; index++)
          sb += String::Format("{0,6} {1,15:N0}\n",
                              years[index], population[index]);

Also, there should be a `using namespace System::Text` statement at the beginning of the file, and the variable name in the `Console::WriteLine` method call should be changed,